docs/docs/general/options.md

```diff
@@ -45,5 +45,8 @@ The context object contains the following properties:
 - `dataset`: dataset at index `datasetIndex`
 - `datasetIndex`: index of the current dataset
 - `active`: true if element is active (hovered)
+- `scale`: the associated scale
```

Comment:
I think I'd document these just on the page where the callback function is mentioned rather than on the general page because they only apply to that one function